packages/nimble-components/src/rich-text/editor/styles.ts

```diff
@@ -188,6 +189,26 @@ export const styles = css`
         color: ${controlLabelDisabledFontColor};
     }
 
+    .ProseMirror a {
+        color: ${linkFontColor};
+        white-space: normal;
+        ${
+            /**
+             * Setting 'pointer-events: none;' to restrict the user from opening a link using the right-click context menu: If the user manually edits
+             * the link's text content, the 'href' attribute of the anchor tag will not be updated. If they attempt to open it using
+             * the right-click context menu with 'Open in new tab/window,' it will still navigate to the link specified
+             * in the 'href' attribute, which may create unnecessary confusion while trying to open the link.
```

Comment:
What's going to fix this behavior? Do we need to submit a bug to prose mirror / tiptap? Is there an associated nimble issue to link to?

Reply:
Discussed with the code owners and the two things that seemed important were to capture the tech debt being introduced to workaround hyperlink issues in an issue which was done and in source for all the workarounds that were added to include a link to the issue that when resolved should remove them.

I went through and placed a comment on all the code that seems like workarounds / tech debt added for the absolute url only case that will be cleaned up / removed when full hyperlink support is added. Please make sure all of them are commented. 👍